### PR TITLE
completed experiment page

### LINKED ISSUE #1302: Completed experiment page
Completed experiments have no enable button and special disable button treatment along with different intro content and other elements, as seen: https://mozilla.invisionapp.com/share/Q77S1H6ZC#/screens/181298065

### PR DESCRIPTION
**Pull request by dannycoates**
fixes #1302 

spec https://mozilla.invisionapp.com/share/Q77S1H6ZC#/screens/181298065

### CODE REVIEW

**Comment by dannycoates** [comment]
### Not enabled

screen shot 2016-10-17 at 1 45 53 pm

### Enabled

screen shot 2016-10-17 at 1 50 57 pm

### Disable dialog

screen shot 2016-10-17 at 1 51 26 pm

**Comment by dannycoates** [comment]
@johngruen a couple differences from the spec...
- I left the stats section visible. I think it's useful info even after retirement.
- I left out the text about "we're working on a report..." which I think should go into the introduction section if necessary.

thoughts?

**Comment by johngruen** [comment]
@dannycoates looks good to me, maybe we could make the string in that red box something served from the YAML so it's customizable depending on the experiment? I could see us wanting a bit of flexibility here.

**Comment by dannycoates** [comment]
> maybe we could make the string in that red box something served from the YAML so it's customizable depending on the experiment?

like all the text in the box (with formatting) or just the body below the "This experiment has been retired..." heading?

I'd rather have the "scare box" text as short as possible and put any additional info below it. What's the minimum we can say? Or do we even _need_ it at all? If the intro text is changed on the retire date it should be pretty clear the experiment isn't available anymore without the scare box. crazy???

**Comment by johngruen** [comment]
@dannycoates maybe we don't need it at all? Keep the class in sass, so we can add whatever we need in yaml

File: frontend/src/app/reducers/addon.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export const getInstalled = (state) => state.installed;
 export const isExperimentEnabled = (state, experiment) =>
   !!(experiment && experiment.addon_id in state.installed);
 
-export const isExperimentCompleted = (experiment) =>
+export const isAfterCompletedDate = (experiment) =>
```
Comment (dannycoates) [inline frontend/src/app/reducers/addon.js:40]: I decided to change the name of this after I wrote a line like `if(!experiment.completed || isExperimentCompleted(experiment))` 😳

File: frontend/src/app/containers/ExperimentPage.js [hunk 2]
```diff
@@ -438,16 +437,26 @@ export class ExperimentDetail extends React.Component {
...
+        <span data-l10n-id="completedDateLabel" data-l10n-args={JSON.stringify({ completedDate })}>Experiment End Date: {completedDate}</span>
+      );
+    }
+    if (!installation_count || installation_count <= 100) {
       return (
         <span data-l10n-id="userCountContainerAlt" className="bold" data-l10n-args={JSON.stringify({ title })}>
         Just launched!</span>
       );
     }
+    const installCount = installation_count.toLocaleString();
```
Comment (dannycoates) [inline frontend/src/app/containers/ExperimentPage.js:456]: I couldn't help but change this while I was here -- should fix #1594

**Comment by dannycoates** [comment]
no boxes
screen shot 2016-10-19 at 10 32 10 am
screen shot 2016-10-19 at 10 33 19 am

File: content-src/experiments/universal-search.yaml [hunk 3]
```diff
@@ -95,4 +95,5 @@ contributors:
 installation_count: 39943
 created: '2016-04-28T03:57:32.270681Z'
 modified: '2016-07-20T15:42:28.873321Z'
+completed: '2016-10-01T15:42:28.873321Z'
```
Comment (dannycoates) [inline content-src/experiments/universal-search.yaml]: remove me before merge

**Comment by dannycoates** [comment]
and ready for code review

**Comment by lmorchard** [comment]
Taking a look...

**Review by lmorchard — APPROVED** [review]
Tests pass, code looks good, works for me!

@dannycoates - Going to leave it un-merged, though, so you can snip out that completed date from the YAML and self-merge

**Pull request merged**